src/components/forms/groups/Create.js

```diff
@@ -3,11 +3,61 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { reduxForm, Field, stopSubmit } from 'redux-form';
 
-import Input from 'components/Input';
+import styles from './styles/index.scss';
 import Button from 'components/Button';
 
 const CREATE_GROUP_FORM = 'createGroup';
 
+const renderInputField = ({ input, label, placeholder, meta: { touched, error } }) => (
+  <div>
+    <div>
+      <label>{label}</label>
+    </div>
+    <input className="form-control" {...input} placeholder={placeholder} />
+    {touched && <div className={`text-danger ${styles.error}`}>{error}</div>}
+  </div>
```

Comment:
I think it's a wrong abstraction because it tries to abstract all kinds of form fields on a single component, so having way too many responsibilities and not following the Open Closed Principle. See a section 2 in http://thefullstack.xyz/solid-javascript/, thoughbot's explanation: https://robots.thoughtbot.com/back-to-basics-solid#openclosed-principle or a very in-depth discussion about it https://8thlight.com/blog/uncle-bob/2014/05/12/TheOpenClosedPrinciple.html